NEWS.md

```diff
@@ -1,104 +1,136 @@
-3.28.1
+3.29.1
 ======
 
 ## Highlights
 
- * `http`: add support for proxy option
-
-   Example:
-   ```
-   log {
-      source { system(); };
-      destination { http( url("SYSLOG_SERVER_IP:PORT") proxy("PROXY_IP:PORT") method("POST") ); };
-   };
-   ```
-   ([#3253](https://github.com/syslog-ng/syslog-ng/pull/3253))
+<Fill this block manually from the blocks below>
```

Comment:
Do you want to add highlights for this release? `panos-parser()` or PCRE dupnames might be interesting.

Reply:
could you open a sub-pr for this? meanwhile I'm working on the release tarball & packages.

Reply:
Yeah, I'm going to open a PR against your fork. :)